buildlib/pr/io_demo/io-demo.yml

```diff
@@ -122,59 +39,78 @@ jobs:
           pathToPublish: '$(Build.ArtifactStagingDirectory)'
           artifactName: drop_$(Build.BuildId)
 
-  - job: test
-    dependsOn: io_build
-    workspace:
-      clean: all
-
-    pool:
-      name: MLNX
-      demands: ${{ parameters.demands }}
-
-    strategy:
-      matrix:
-        ${{ each test in parameters.tests }}:
-          ${{ test.Key }}:
-            test_name: ${{ test.Value.test_name }}
-            test_args: ${{ test.Value.args }}
-            test_time: ${{ test.Value.duration }}
-            test_intefrafe: ${{ test.Value.interface }}
-            test_ucx_tls: ${{ test.Value.tls }}
-            test_extra_run_args: ${{ test.Value.extra_run_args }}
-            initial_delay: ${{ coalesce(test.Value.initial_delay, parameters.initial_delay) }}
-            test_interference: ${{ test.Value.interference }}
-      maxParallel: 1
+  - template: jobs-io-demo.yaml
+    parameters:
+      demands: "ucx_iodemo -equals yes"
```

Comment:
maybe call this tag "ucx_iodemo_exclusive"

Reply:
disagree,  what's "exclusive" - no additional information ( but for CI need rename all labels in code and on hosts)

Reply:
ok